hooks/src/index.js

```diff
@@ -268,6 +281,10 @@ function invokeEffect(hook) {
 	if (typeof result === 'function') hook._cleanup = result;
 }
 
+/**
+ * @param {any[]} oldArgs
+ * @param {any[]} newArgs
+ */
 function argsChanged(oldArgs, newArgs) {
 	return !oldArgs || newArgs.some((arg, index) => arg !== oldArgs[index]);
```

Comment:
I know it's not part of this PR, but I wonder if this is smaller after gzip?

```suggestion
    if (!oldArgs || oldArgs.length!==newArgs.length) return;
    for (let i=newArgs.length; i--; ) if (newArgs[i]!==oldArgs[i]) return;
    return true;
```

Reply:
No worries. All comments welcome 😄 

This function returns `true` if the args are different so I think the `return` values need to be swapped like so:

```suggestion
	if (!oldArgs || oldArgs.length!==newArgs.length) return true;
	for (let i=newArgs.length; i--; ) if (newArgs[i]!==oldArgs[i]) return true;
	return;
```

In my tests it adds 21 B. However, given that `argsChanged` is potentially run a lot in `render` methods, perhaps it might be worth the bytes for the perf boost? (I'm assuming a `for` loop is marginally faster than `.some`).